website/src/content/development/data-fetching.mdx

```diff
@@ -279,3 +279,27 @@ The available proxy targets are exposed as a `MC_API_PROXY_TARGETS` constant in
 The SDK library does not include features like data normalization, caching, etc. You will need to build these on your own.
 
 </Warning>
+
+<!-- TODO: ### Using your own HTTP client -->
+
+### Using file uploads
+
+In case you need to deal with file uploads, you will probably use [`FormData`](https://developer.mozilla.org/en-US/docs/Web/API/FormData).
+
+The `FormData` object can then be sent as the request HTTP body.
+
+One important thing is to properly configure the `Content-Type` HTTP header, which usually must be set to `multipart/form-data`. However, you also need to provide a `boundary` [directive](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type#directives).
+
+The best way to properly configure it, is to let the browser do it. You do so by not providing the `Content-Type` HTTP header in the request. The browser will automatically set one.
```

Comment:
In this case the browser can infer the Content-Type correctly. We allow the browser to do so by not...

Then omit last sentence. 

Sorry am on my phone.